### ISSUE TITLE
Go get deprecated, build instructions out of date

### ISSUE THREAD

**Issue by gamedevsam** [probonopd/go-appimage#230]
I attempted to follow instructions here: https://github.com/probonopd/go-appimage/blob/master/src/appimaged/README.md#building

And ran into the following error:
```
go get github.com/probonopd/go-appimage/src/appimaged
go: go.mod file not found in current directory or any parent directory.
	'go get' is no longer supported outside a module.
	To build and install a command, use 'go install' with a version,
	like 'go install example.com/cmd@latest'
	For more information, see https://golang.org/doc/go-get-install-deprecation
	or run 'go help get' or 'go help install'.
```

**Comment by gamedevsam** [comment]
For anyone interested in building this yourself, it's as simple as:

```
git clone https://github.com/probonopd/go-appimage
cd go-appimage
./scripts/build.sh -a amd64
```

**Comment by CalebQ42** [comment]
It looks like the individual tool's READMEs have just been neglected for a bit. It would probably be best if you submitted a PR to fix them.

As a note, you only have to specify `-a` in the build script if you're building for non-native architectures (and the build script currently only works on amd64). Also the individual READMEs should only build for the current tool so it would be more correct as `scripts/build.sh appimaged`.

**Comment by parkerlreed** [comment]
zig download gives a 403 forbidden, halting the building process.

```
(1)(deck@steamdeck go-appimage)$ ./scripts/build.sh -a amd64
+ '[' 2 -gt 0 ']'
+ case $1 in
+ BUILDARCH=(${2//,/ })
+ shift
+ shift
+ '[' 0 -gt 0 ']'
+ '[' '!' -z '' ']'
++ date +%Y-%m-%d_%H%M%S
+ export COMMIT=2022-12-16_141722
+ COMMIT=2022-12-16_141722
++ date +%Y-%m-%d_%H%M%S
+ export VERSION=2022-12-16_141722
+ VERSION=2022-12-16_141722
+ '[' ']'
+ '[' -z ']'
+ BUILDTOOL=(appimaged appimagetool mkappimage)
+ '[' -z amd64 ']'
+ '[' ']'
+++ dirname ./scripts/build.sh
++ cd ./scripts
++ pwd
+ PROJECT=/home/deck/build/go-appimage/scripts/..
+ '[' -z ']'
+ BUILDDIR=/home/deck/build/go-appimage/scripts/../build
+ '[' '!' -z ']'
+ mkdir -p /home/deck/build/go-appimage/scripts/../build
+ cd /home/deck/build/go-appimage/scripts/../build
+ '[' '!' -e /home/deck/build/go-appimage/scripts/../build/zig ']'
+ wget -c -q https://ziglang.org/builds/zig-linux-x86_64-0.10.0-dev.2112+0df28f9d4.tar.xz
(1)(deck@steamdeck go-appimage)$ wget -c -q https://ziglang.org/builds/zig-linux-x86_64-0.10.0-dev.2112+0df28f9d4.tar.xz
(8)(deck@steamdeck go-appimage)$ wget -c https://ziglang.org/builds/zig-linux-x86_64-0.10.0-dev.2112+0df28f9d4.tar.xz
--2022-12-16 14:18:37--  https://ziglang.org/builds/zig-linux-x86_64-0.10.0-dev.2112+0df28f9d4.tar.xz
Loaded CA certificate '/etc/ssl/certs/ca-certificates.crt'
Resolving ziglang.org (ziglang.org)... 18.244.202.56, 18.244.202.54, 18.244.202.107, ...
Connecting to ziglang.org (ziglang.org)|18.244.202.56|:443... connected.
HTTP request sent, awaiting response... 403 Forbidden
2022-12-16 14:18:37 ERROR 403: Forbidden.

```

**Comment by CalebQ42** [comment]
If you replace the URL with https://ziglang.org/download/0.10.0/zig-linux-x86_64-0.10.0.tar.xz it should work. Fixed in #235

**Comment by parkerlreed** [comment]
Thank I noticed it also worked with the lastest master tar.gz as well
